docs/developer/example-configurations/renkulab.rst

```diff
@@ -0,0 +1,98 @@
+
+    http://<your-minikube-ip>/auth/realms/Renku/broker/renkulab/endpoint
+    http://<your-minikube-ip>/api/auth/renkulab/token
+    http://<your-minikube-ip>/api/auth/jupyterhub/token
+    http://<your-minikube-ip>/jupyterhub/hub/oauth_callback
+
+Step 2
+--------
+Copy the file :code:`charts/example-configurations/minikube-values-renkulab-template.yaml`
+to :code:`charts/example-configurations/minikube-values-renkulab.yaml`. Complete the
+:code:`minikube-values-renkulab.yaml` file by replacing :code:`# Put Application Id here!`
+with in the Application Id from the Renku-local application created in Step 1 and
+:code:`# Put Application Secret here!` with the Secret created in Step 1.
+
+Now start the Renku platform following the steps described in the `Renku charts README`_,
+replacing the final :code:`helm upgrade` command with the following:
+
+.. _`Renku charts README`: https://github.com/SwissDataScienceCenter/renku/blob/master/charts/README.rst
+
+.. code-block:: console
+
+  $ helm upgrade renku --install --namespace renku \
+    -f example-configurations/minikube-values-renkulab.yaml \
+    --set global.renku.domain=$(minikube ip) \
+    --set ui.jupyterhubUrl=http://$(minikube ip)/jupyterhub \
+    --set ui.gatewayUrl=http://$(minikube ip)/api \
+    --set gateway.keycloakUrl=http://$(minikube ip) \
+    --set notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://$(minikube ip)/api/auth/jupyterhub/token \
+    ./renku
+
+Step 3
+--------
+Open :code:`http://<your-minikube-ip>/auth` in your browser and login to the
+admin console using admin/admin. Under Identity Providers choose "OpenID Connect" from
+the "Add Provider..." drop-down menu. Enter the following values into the form:
+
++-------------------+--------------------------------------------+
+| Field             | Value                                      |
++===================+============================================+
+| Alias             | renkulab                                   |
++-------------------+--------------------------------------------+
+| Authorization URL | https://renkulab.io/gitlab/oauth/authorize |
++-------------------+--------------------------------------------+
+| Token URL         | https://renkulab.io/gitlab/oauth/token     |
++-------------------+--------------------------------------------+
+| Client ID         | [Application Id from Step 1]               |
++-------------------+--------------------------------------------+
+| Client Secret     | [Secret from Step 1]                       |
++-------------------+--------------------------------------------+
+
+
+Save the identity provider and logout from the Keycloak admin panel.
+
+Step 4
+--------
+For each created project you will have to modify the :code:`.gitlab-ci.yaml`
+in the repository for the image build to work. In the image_build job
+
+- remove the :code:`image-build` tag
+- add the following entry:
+
+.. code-block:: console
+
+  services:
+    - docker:dind
```

Comment:
does this actually work with the runner configured on renkulab.io?

Reply:
Need to test -- I had copied this section from the gitlab.com configuration documentation,